### DISCUSSION TITLE
Problems with the favicon.ico

### DISCUSSION THREAD

**Discussion by javiermaciasa** [vercel/next.js#48590] [Help]
### Summary

I've been attempting to compile my code but keep encountering an error regarding a supposedly missing "favicon.ico," even though it's present and is the original one provided in the Nextjs 13 App Playground. Has anyone else experienced this issue? I need to upload my project, but this error is preventing me from doing so.
I also added the favicon.ico in the public folder.
This is the error:
![image](https://user-images.githubusercontent.com/79488329/233179247-3a61997a-bda7-46f4-a3c1-4596c537eb0d.png)

The structure of the project: 
![image](https://user-images.githubusercontent.com/79488329/233179586-05ba7ddc-438b-4e6e-a855-47e02798ef2f.png)

**Comment by haydenbleasel** [comment]
@javiermaciasa While this isn't a direct solution to the problem you're having, I noticed you're using the new App directory so you may benefit from implementing [file-based metadata](https://beta.nextjs.org/docs/api-reference/metadata#app-icons) to solve the favicon issue.

**Reply by javiermaciasa** [reply]
Thanks, I will try that solution, however what is weird is that I'm using the same tamplate that Nextjs provide for trying nextjs13
